Cargo.toml

```diff
@@ -23,8 +23,8 @@ eq-common = { path = "common", version = "0.1.8" }
 eq-sdk = { path = "sdk", version = "0.1.8" }
 
 clap = "4.5"
-celestia-types = "0.11"
-celestia-rpc = "0.11"
+celestia-types = "0.12"
+celestia-rpc = "0.11.2"
```

Comment:
I suggest we do not specify the patch. Rather, we update the lock from time to time, like I do in #92 
I also bumped in https://github.com/celestiaorg/eq-service/pull/92/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R26 and think we should move to merge that instead of just this bump